chat.css

```diff
@@ -1,309 +1,328 @@
+  font-size: 150%;
+}
+
+.message {
+  overflow: auto;
+}
+
+.typing {
+  color: gray;
+  overflow: auto;
+}
+
+.username {
+  font-weight: 700;
+  padding-right: 15px;
+  text-align: right;
+}
+
+/* Input */
+
+#inputMessage {
+  font-size: 100%;
+  bottom: 0;
+  height: auto;
+  left: 0.5%;
+  outline: none;
+  border: 1px solid #40444b;
+  padding-left: 10px;
+  position: absolute;
+  right: 0;
+  width: 84.5%;
+  overflow-y: auto;
+  resize: none;
+  border-radius: 8px;
+  max-height: 144px;
+}
+
+.lightThemeScrollbar::-webkit-scrollbar-track
+{
+  -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0.3);
+  border-radius: 10px;
+  background-color: #fff;
+}
+.lightThemeScrollbar::-webkit-scrollbar
+{
+  width: 12px;
+  border-radius: 10px;
+  background-color: #fff;
+}
+
+.lightThemeScrollbar::-webkit-scrollbar-thumb
+{
+  border-radius: 10px;
+  -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0.3);
+  background-color: #555;
+}
+
+.darkThemeScrollbar::-webkit-scrollbar-track
+{
+  -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0.3);
+  border-radius: 10px;
+  background-color: #555;
+}
+.darkThemeScrollbar::-webkit-scrollbar
+{
+  width: 12px;
+  border-radius: 10px;
+  background-color: #555;
+}
+
+.darkThemeScrollbar::-webkit-scrollbar-thumb
+{
+  border-radius: 10px;
+  -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0.3);
+  background-color: #202225;
+}
+
+/* User List */
+
+#userList {
+  border-left: 3px solid #40444b;
+  bottom: 0;
+  height: 100%;
+  outline: none;
+  padding-left: 10px;
+  position: absolute;
+  right: 0;
+  width: 14.5%;
+  top: 0px;
+}
+
+/* Users in user list */
+.user {
+  overflow: auto;
+  display: list-item;
+  padding-top: 5px;
+  padding-bottom: 5px;
+}
+
+.messageBody p {
+  display: inline;
+}
+
+#notificationBell {
+  margin: 10px;
+  position: absolute;
+  top: 0px;
+  right: 50px;
+  height: 50px;
+  width: 50px;
+  cursor: pointer;
+}
+
+.settingsIcon {
+  margin: 10px;
+  position: absolute;
+  top: 0px;
+  right: 0px;
+  height: 50px;
+  width: 50px;
+  cursor: pointer;
+}
+
+#typingMessageArea {
+  bottom: 5%;
+  height: 5%;
+  position: absolute;
+  width: 84.5%;
+  margin: 0;
+  padding: 10px 20px 10px 20px;
+  font-size: 150%;
+}
+
+#reconnectingPage {
+  display: none;
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+}
+
+#reconnectingArea {
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+#reconnectingIcon {
+  background-image: url("./assets/Reconnecting.svg");
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  width: 30%;
+  height: 30%;
+}
+
+body .light, .light #inputMessage, .light #pages {
+  background-color: #fff;
+  color: #212529
+}
+
+body .dark, .dark #inputMessage, .dark #pages {
+    background-color: #36393f;
+    color:#fff;
+}
+
+li[id$="Page"] {
+  display: none;
+  transition: display 0.5s, opacity 0.5s linear;
+}
+
+li#loginPage {
+  display: block;
+}
```

Comment:
I can't tell exactly what your changing here, can you explain? What's up with the last four lines?
```css
body .light, .light #inputMessage, .light #pages {
  background-color: #fff;
  color: #212529
}

body .dark, .dark #inputMessage, .dark #pages {
    background-color: #36393f;
    color:#fff;
}

li[id$="Page"] {
  display: none;
  transition: display 0.5s, opacity 0.5s linear;
}

li#loginPage {
  display: block;
} 
```